src/core/core.js

```diff
@@ -242,7 +243,15 @@ var Page = (function PageClosure() {
                                                     self.fontCache);
 
         return partialEvaluator.getTextContent(contentStream,
-                                               self.resources);
+                  self.resources).then(function (data) {
+          var mbox = self.getPageProp('MediaBox');
+          var bounds = {y : mbox[0], x : mbox[1],
+                        width : mbox[2], height : mbox[3]};
```

Comment:
Nit: unfortunately the spacing isn't completely correct here now, but I seem to recall that it was correct at one previous point in time :-)
This should be:
```
var bounds = { y: mbox[0], x: mbox[1],
               width: mbox[2], height: mbox[3] };
```
Unfortunately this seems to apply in numerous places below as well.